atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsDeleteBenchmarks.java

```diff
@@ -36,6 +39,18 @@ private Object doDelete(RegeneratingTable<Multimap<Cell, Long>> table) {
         return table.getTableCells();
     }
 
+    private Object doDeleteRange(ConsecutiveNarrowTable table, int numBatches) {
+        Iterable<RangeRequest> rangeRequests;
```

Comment:
That used to specifically be against our coding standards, because ternary can make code more compact in the harder-to-read-way. If it's not anymore, I'll use ? all the time! 😺